spec/abi.dd

```diff
@@ -206,12 +206,14 @@ $(H3 $(LNAME2 name_mangling, Name Mangling))
     $(GNAME MangledName):
         $(B _D) $(GLINK QualifiedName) $(GLINK Type)
         $(B _D) $(GLINK QualifiedName) $(B M) $(GLINK Type)
+        $(B _D) $(GLINK QualifiedName) $(B Z)        $(GREEN // Internal)
 
     $(GNAME QualifiedName):
         $(GLINK SymbolName)
         $(GLINK SymbolName) $(I QualifiedName)
 
     $(GNAME SymbolName):
+        $(B 0)             $(GREEN // anonymous symbols)
         $(GLINK LName)
         $(GLINK TemplateInstanceName)
```

Comment:
Amended.

BTW: I've only seen this with overload sets as template arguments, but it looks like a bug to me: the overloaded symbol has these anonymous parents, but something like the location of the lookup should be encoded instead.